cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -35,7 +35,7 @@
     public static final int NO_ANIMATION = 0;
     
     public interface OnDismissListener {
-        void onDismiss();
+        void onDismiss(boolean timeout);
```

Comment:
I'll add listener rather than method as adding another one will complain on the existing implementation.